README.md

```diff
@@ -40,3 +65,68 @@ export NVM_DIR="$HOME/.nvm"
+
+### Which Branch?
+
+The `develop` branch acts as a testnet containing the latest changes. The `main` branch is production branch for the **app.sovryn.com**. Depending on your feature you are contributing with, select the proper branch as a starting point. Most of the time, it will be the `develop` branch unless you provide hotfixes or features that should be released before other features - then it can be `main`. By doing so, we merge all features to `develop` and then `develop` to `main` to make one big release batch, after full testing and review.
+
+**All** bug fixes should be sent to the latest stable `main` branch. Bug fixes should never be sent to the development branch unless they fix features that exist only in the upcoming release.
+
+**Minor** features that are fully backward compatible with the current release may be sent to the latest stable branch.
+
+**Major** new features should always be sent to the `develop` branch, which contains the upcoming release.
+
+Ask in the `#technical-discussion` channel of the Sovryn Discord server when unsure if the feature qualifies as major or minor.
+
+### Working With Sovryn UI LIbrary
+
+All UI components for general use are located in `@sovryn/ui` package. Each component must include a Storybook example and Jest tests that verify functionality is working as intended. Components should not include implementation specific details (e.g. contract addresses, complex state, hardcoded text), and instead expose these via component props. Figma designs for all components, and general design guidelines that we followed when constructing this library, can be found [here](https://www.figma.com/file/Ig2ZfR16Svs8In7yibukrO/Sovryn-UI-Library).
+
+### Storybook
+
+We use Storybook to provide API style docs and examples for our UI components. Any new components should have stories added to them that include all available properties that can be passed in, and some specific examples for major common variations. For specific implementation examples, you can search the codebase for filenames ending in `.stories.tsx`.
+
+To test components via Storybook on your local machine:
+
+- run `yarn install` on your working branch.
+- run `yarn storybook` to load the development server.
+- access Storybook dev server on `localhost:6006`.
+- if the docs fail to load and you see an error `cannot read properties of undefined (reading 'storyStore')`, the package installation is likely broken and you will need to clear `node_modules` folder and follow first two steps again.
+- changes can now be made to `index.stories.tsx` files and the development server will hot-reload automatically.
```

Comment:
```suggestion
- changes can now be made to `[ComponentName].stories.tsx` files and the development server will hot-reload automatically.
```
We don't use `index.stories.tsx` anymore.

Reply:
whoops, forgot to remove both references - updated to make it consistent with other reference to this filename